Portfolio/index.html

```diff
@@ -1,16 +1,116 @@
+                    <p>Name = Aayush SETIA<br>
+                    experience= 2 years<br>
+                    Age=20</p>
+                </details>
+                <h2>SKILLES<H2/>
+                <i class="fa-brands fa-html5"></i>
+                <i class="fa-brands fa-css3-alt"></i>
+                <i class="fa-brands fa-js"></i>
+                <i class="fa-brands fa-react"></i>
+                <div class="container">
+                    <h2>connect with us </h2>
+                    <p> we would love to respond to your queries and help you</p>
+                    <div class="contact-box">
+                        <div class="contact-left">
+                            <h3> Send Your Request</h3>
+                            <form><div class="input-row">
+                                    <div class="input-group">
+                                        <label>Name</label>
+                                        <input type="text" placeholder="Aayush">
+                                        </div>
+                                        <div class="input-group">
+                                            <label>Phone number</label>
+                                            <input type="text" placeholder="+916280083340">
+                                            </div>
+                                            <div class="input-row">
+                                    <div class="input-group">
+                                        <label>email</label>
+                                        <input type="email" placeholder="aayushsetia@gmail.com">
+                                        </div>
+                                        <div class="input-group">
+                                            <label>subject</label>
+                                            <input type="text" placeholder="Anything">
+                                            </div>
+                                            </div>
+                                            <label> message</label>
+                                            <textarea rows="5" placeholder="your message
+                                            "></textarea>
+                                            <button type="submit">Send</button>
+                                            </form>
+                                            </div>
+                                            </body>
+                                            </main>
+                                            <div class="contact-right">
+                                                <h3> Reach Us </h3>
+                                                <table>
+                                                    <tr>
+                                                        <td>email</td>
+                                                        <td>contactus@gmail.com</td>
+                                                        </tr>

```

Comment:
There is an unnecessary closing quotation mark at the end of the line that should be removed.